Sources/BottomSheet/BottomSheet.swift

```diff
@@ -50,11 +53,23 @@ public extension View {
             sheetContent: sheetContent
         )
     }
-
+    
+    /// Presents a sheet.
+    /// - Parameter sheetContent: A closure that returns the content of the sheet.
+    func bottomSheet<SheetContent: View>(
+        @ViewBuilder sheetContent: () -> SheetContent
+    ) -> BottomSheet<Self, SheetContent> {
+        BottomSheet(
+            isPresented: .constant(true),
+            content: { self },
+            sheetContent: sheetContent
+        )
+    }
 }
 
 private struct ExampleView: View {
     let colors: [Color] = [.red, .orange, .yellow, .green, .blue, .indigo, .purple]
+    @State private var isShowingSheet = false
```

Comment:
After implementing previous suggestion, this can then be passed into the `bottomSheet`

Reply:
Removed this @State from this example.